PokeAlarm/Events/Gym.py

```diff
@@ -34,6 +34,10 @@ def __init__(self, data):
             str, data.get('description'), Unknown.REGULAR).strip()
         self.gym_image_url = check_for_none(
             str, data.get('url'), Unknown.REGULAR)
+        self.slots_available = check_for_none(
+            int, data.get('slots_available'), Unknown.TINY)
+        self.guard_count = check_for_none(
+            int, 6 - data.get('slots_available'), Unknown.TINY)
```

Comment:
I think this might throw a value error if 'slots_available' is None. You may want to test to confirm - if so, the following might be a good alternative:
```python
self.guard_count = (
    (6 - self.slots_available) if Unknown.is_not(self.slots_available)
    else Unknown.TINY),
```

Reply:
Are the new events in use at this point or still relying on old event structure.

Reply:
The old structure is still in place.

Reply:
I'll test this tonight.